packages/admin/resources/lang/pl/pages/dashboard.php

```diff
@@ -2,6 +2,6 @@
 
 return [
 
-    'title' => 'Dashboard',
```

Comment:
I didn't translate this phrase because it is known in Polish. And the translation "Pulpit" can be misleading, because "pulpit" = "desktop", and the "dashboard" = "deska rozdzielcza" or "tablica rozdzielcza". Is not used on the Internet, it is more common in the automotive industry